### DISCUSSION TITLE
Purple Pi switched to KaTeX

### DISCUSSION THREAD

**Discussion by nschloe** [KaTeX/KaTeX#3011] [Show and tell]
Just a small heads-up that I migrated [Purple Pi](https://github.com/nschloe/purple-pi) to KaTeX this afternoon, and I'm more than happy with it. Purple Pi is a browser extension that supercharges the web with LaTeX, e.g., GitHub pages, GitHub Wikis etc. Even this page right here!
$$\int_a^b f'(x) dx = f(b) - f(a)$$
(One just needs to include an [activation link](https://github.com/nschloe/purple-pi?activate) somewhere in the page.)

**Comment by edemaine** [comment]
This is great!  I've wanted a KaTeX browser extension.

I just tried installing Purple Pi but it seems to render MathJax.  Perhaps not released yet?  

When released, you could add it to our [Users page](https://katex.org/users.html). Or I think it might be appropriate for [Extensions & Libraries](https://katex.org/docs/libs.html).

**Comment by nschloe** [comment]
Ah yes, the release process is slow. It should be out now, version 0.3.0. It kind of fits on either one of your suggestions. Which  so you think is more appropriate?

**Reply by edemaine** [reply]
KaTeX version working now! ❤️ 

Maybe we should add a "Chrome extensions" category to Extensions & Libraries. In the same section, we could link to [Aaron Choo's KaTeX GitHub Chrome Extension](https://github.com/AaronCQL/katex-github-chrome-extension) that you helpfully point to from Purple Pi's README.
